onboarding.md

```diff
@@ -30,26 +30,22 @@ That is it! Go to your onboarding issue and start checking things off!
 
 A member of the [PeopleOps] team will do the following to start the onboarding process:
 
-* Add the new team member to the [Trialists] group, allowing access to [operations](https://github.com/niteoweb/operations) and [handbook](https://github.com/niteoweb/handbook) repositories.
-* Create a new [Trialist Onboarding Issue] and assign it to the new team member.
 * Send the <a href="mailto:?
 subject=Welcome to Niteo
 &body=
-Hey!%0D%0A
+Hey, welcome to Niteo!%0D%0A
 %0D%0A
-Welcome to Niteo! We keep everything related to work on GitHub so I will be short:%0D%0A
+We keep everything related to work on GitHub so I will be short:%0D%0A
 %0D%0A
-• First, please read https://github.com/niteoweb/handbook/blob/master/onboarding.md%0D%0A
-• Then go to LINK TO ONBOARDING ISSUE and start at the top.%0D%0A
+• Firstly, please read https://github.com/niteoweb/handbook/blob/master/onboarding.md%0D%0A
+• And reply with your GitHub handle.%0D%0A
 %0D%0A
-%0D%0A
-Regards%0D%0A
-%0D%0A
-" target="_blank">
-Welcome to Niteo
-</a>email.
+Regards%0D%0A%0D%0A
+">Welcome to Niteo email</a> and the request for their GitHub handle.
+* Add the new team member to the [Trialists] group, allowing access to [operations](https://github.com/niteoweb/operations) and [handbook](https://github.com/niteoweb/handbook) repositories.
```

Comment:
This needs to be the first task, otherwise when you send the email, the trialist won't be able to access their onboarding ticket.

Reply:
@zupo The content of the email changed. Me and @cas-- did the changes based on my issues with the process when I onboarded the two new designers.

Reply:
Got it, sry for the noise.